Tests/CucumberMessages.CompatibilityTests/CucumberMessages.Tests.csproj

```diff
@@ -0,0 +1,262 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The CucumberMessages.Tests project doesn't have a "Reqnroll" prefix. Is this wanted?

Reply:
Project renamed to Reqnoll.CucumberMessages.Tests.